src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/FindOneTableIntegrationTest.java

```diff
@@ -87,6 +87,17 @@ public void findOneSingleStringKey() {
                           """;
       insertOneInTable(TABLE_WITH_STRING_ID_AGE_NAME, DOC_B_JSON);
 
+      // Third one with missing "age" and null "name"
```

Comment:
Turns out 'text' and 'int' codecs happened to work with `null`s, although for different reasons. But was simple and cheap to add coverage here.